eslint/README.md

```diff
@@ -0,0 +1,105 @@
+# [ESL](../../../) ESL migration support
+
+Version: *4.12.0*
```

Comment:
```suggestion
```
We should skip it here or manage it indirectly.
Otherwise, we have one more point for extra version update behaviors...